core/server/update-check.js

```diff
@@ -38,6 +38,24 @@ var crypto   = require('crypto'),
     checkEndpoint = 'updates.ghost.org',
     currentVersion = config.ghostVersion;
 
+/**
+ * Attempts to parse string to JSON.
+ * @return {Object} is successful. Otherwise, undefined.
+ */
+function safeParseJson(json) {
```

Comment:
Just a comment: personally I'd be against monkey-patching the JSON object, especially for those that use Ghost as a NPM module, however I agree with @kirrg001